src/App.vue

```diff
@@ -0,0 +1,25 @@
+<template>
+  <header><NavBar /></header>
+  <router-view/>
+  <div class="container">
+  
+  </div>
+</template>
+<script>
```

Comment:
Remove this section if not in use. Always avoid having code that doesn't do anything or isn't used. This will avoid technical debts.